2021-04-01 20:07:34,637 INFO    MainThread:20160 [wandb_setup.py:_flush():69] setting env: {}
2021-04-01 20:07:34,637 INFO    MainThread:20160 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-01 20:07:34,637 INFO    MainThread:20160 [wandb_init.py:_log_setup():319] Logging user logs to C:\Users\neilg\PycharmProjects\DL\A2\partA\wandb\run-20210401_200734-3lzqamaz\logs\debug.log
2021-04-01 20:07:34,638 INFO    MainThread:20160 [wandb_init.py:_log_setup():320] Logging internal logs to C:\Users\neilg\PycharmProjects\DL\A2\partA\wandb\run-20210401_200734-3lzqamaz\logs\debug-internal.log
2021-04-01 20:07:34,638 INFO    MainThread:20160 [wandb_init.py:init():352] calling init triggers
2021-04-01 20:07:34,638 INFO    MainThread:20160 [wandb_init.py:init():359] wandb.init called with sweep_config: {}
config: {'data_aug': True, 'batch_size': 64, 'n_filters': 32, 'filter_org': 2, 'dropout': 0.2, 'batch_norm': True, 'activation': 'relu'}
2021-04-01 20:07:34,648 INFO    MainThread:20160 [wandb_init.py:init():401] starting backend
2021-04-01 20:07:34,648 INFO    MainThread:20160 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=spawn, using: spawn
2021-04-01 20:07:34,673 INFO    MainThread:20160 [backend.py:ensure_launched():123] starting backend process...
2021-04-01 20:07:34,794 INFO    MainThread:20160 [backend.py:ensure_launched():128] started backend process with pid: 23616
2021-04-01 20:07:34,795 INFO    MainThread:20160 [wandb_init.py:init():406] backend started and connected
2021-04-01 20:07:34,795 INFO    MainThread:20160 [wandb_init.py:init():446] updated telemetry
2021-04-01 20:07:34,796 INFO    MainThread:20160 [wandb_init.py:init():465] communicating current version
2021-04-01 20:07:36,421 INFO    MainThread:20160 [wandb_init.py:init():470] got version response upgrade_message: "wandb version 0.10.24 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-04-01 20:07:36,421 INFO    MainThread:20160 [wandb_init.py:init():478] communicating run to backend with 30 second timeout
2021-04-01 20:07:36,869 INFO    MainThread:20160 [wandb_init.py:init():503] starting run threads in backend
2021-04-01 20:07:39,206 INFO    MainThread:20160 [wandb_run.py:_console_start():1422] atexit reg
2021-04-01 20:07:39,206 INFO    MainThread:20160 [wandb_run.py:_redirect():1285] redirect: SettingsConsole.WRAP
2021-04-01 20:07:39,207 INFO    MainThread:20160 [wandb_run.py:_redirect():1320] Wrapping output streams.
2021-04-01 20:07:39,207 INFO    MainThread:20160 [wandb_run.py:_redirect():1336] Redirects installed.
2021-04-01 20:07:39,207 INFO    MainThread:20160 [wandb_init.py:init():527] run started, returning control to user process
2021-04-01 20:07:44,674 ERROR   MainThread:20160 [util.py:get_module():168] Error importing optional module plotly.tools
Traceback (most recent call last):
  File "C:\Users\neilg\AppData\Local\Programs\Python\Python37\lib\site-packages\wandb\util.py", line 163, in get_module
    return import_module(name)
  File "C:\Users\neilg\AppData\Local\Programs\Python\Python37\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 953, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'plotly'
2021-04-01 20:07:44,724 INFO    MainThread:20160 [wandb_run.py:_atexit_cleanup():1392] got exitcode: 1
2021-04-01 20:07:44,724 INFO    MainThread:20160 [wandb_run.py:_restore():1364] restore
2021-04-01 20:07:47,938 ERROR   MainThread:20160 [wandb_run.py:_atexit_cleanup():1410] Problem finishing run
Traceback (most recent call last):
  File "C:\Users\neilg\AppData\Local\Programs\Python\Python37\lib\site-packages\wandb\sdk\wandb_run.py", line 1400, in _atexit_cleanup
    self._on_finish()
  File "C:\Users\neilg\AppData\Local\Programs\Python\Python37\lib\site-packages\wandb\sdk\wandb_run.py", line 1537, in _on_finish
    self.history._flush()
  File "C:\Users\neilg\AppData\Local\Programs\Python\Python37\lib\site-packages\wandb\sdk\wandb_history.py", line 59, in _flush
    self._callback(row=self._data, step=self._step)
  File "C:\Users\neilg\AppData\Local\Programs\Python\Python37\lib\site-packages\wandb\sdk\wandb_run.py", line 733, in _history_callback
    row, step, publish_step=not_using_tensorboard
  File "C:\Users\neilg\AppData\Local\Programs\Python\Python37\lib\site-packages\wandb\sdk\interface\interface.py", line 211, in publish_history
    data = data_types.history_dict_to_json(run, data, step=step)
  File "C:\Users\neilg\AppData\Local\Programs\Python\Python37\lib\site-packages\wandb\sdk\data_types.py", line 1980, in history_dict_to_json
    payload[key] = val_to_json(run, key, val, namespace=step)
  File "C:\Users\neilg\AppData\Local\Programs\Python\Python37\lib\site-packages\wandb\sdk\data_types.py", line 2006, in val_to_json
    val = Plotly.make_plot_media(val)
  File "C:\Users\neilg\AppData\Local\Programs\Python\Python37\lib\site-packages\wandb\sdk\data_types.py", line 1930, in make_plot_media
    val = util.matplotlib_to_plotly(val)
  File "C:\Users\neilg\AppData\Local\Programs\Python\Python37\lib\site-packages\wandb\util.py", line 391, in matplotlib_to_plotly
    required="plotly is required to log interactive plots, install with: pip install plotly or convert the plot to an image with `wandb.Image(plt)`",
  File "C:\Users\neilg\AppData\Local\Programs\Python\Python37\lib\site-packages\wandb\util.py", line 170, in get_module
    raise wandb.Error(required)
wandb.errors.error.Error: plotly is required to log interactive plots, install with: pip install plotly or convert the plot to an image with `wandb.Image(plt)`
